contracts/escrow/Escrow.sol

```diff
@@ -163,14 +191,20 @@ contract Escrow {
 	 */
 	function depositTokens(uint256 _amount) public checkStatus(Status.Deposit) {
 		require(_amount > 0, "Amount needs to be bigger than zero.");
+		uint256 amount = _amount;
```

Comment:
either of `amount ` and `_amount` is enough, you don't need both

Reply:
I believe both are required because we need to find the right value.

`_amount` will always contain the total value to be deposited.

Whereas, `amount` will contain either the total value user tries to deposit, or the total value the contract allows (depositLimit).

Reply:
really? i don't see why both would be needed here